diff --git a/src/HypoTestTool.cxx b/src/HypoTestTool.cxx
index 710eed7..1f641a2 100644
--- a/src/HypoTestTool.cxx
+++ b/src/HypoTestTool.cxx
@@ -543,7 +543,7 @@ RooStats::HypoTestTool::SetupHypoTestCalculator(RooWorkspace * w, bool doUL,
         RooCmdArg _InitialHesse(InitialHesse(false));
         RooCmdArg _Hesse(Hesse(false));
         RooCmdArg _Minimizer(Minimizer(mMinimizerType.c_str(), "Migrad"));
-        RooCmdArg _Strategy(Strategy(0));
+        RooCmdArg _Strategy(Strategy(1));
         RooCmdArg _Strategy_alt(Strategy(1));
         RooCmdArg _Verbose(Verbose(verbose));
         RooCmdArg _PrintLevel(PrintLevel(mPrintLevel+1));
diff --git a/src/StatTools.cxx b/src/StatTools.cxx
index f8ef1be..bc1d543 100644
--- a/src/StatTools.cxx
+++ b/src/StatTools.cxx
@@ -341,6 +341,7 @@ nToyRatio            ratio of S+B/B toys (default is 2)
 
     HypoTestTool calc;
     calc.SetParameter("EnableDetailedOutput", true);
+    calc.SetParameter("RandomSeed", 0);
 
     // set parameters
     /*
