Index: src/HypoTestTool.cxx
===================================================================
--- src/HypoTestTool.cxx	(revision 242474)
+++ src/HypoTestTool.cxx	(working copy)
@@ -533,7 +533,7 @@
         tw.Start();
         Bool_t verbose = (m_logger.GetMinLevel() <= kDEBUG) ? kTRUE : kFALSE;
         RooFitResult * fitres = sbModel->GetPdf()->fitTo(*data,InitialHesse(false), Hesse(false),
-                Minimizer(mMinimizerType.c_str(),"Migrad"), Strategy(0), Verbose(verbose),
+                Minimizer(mMinimizerType.c_str(),"Migrad"), Strategy(1), Verbose(verbose),
                 PrintLevel(mPrintLevel+1), Constrain(constrainParams), Save(true) );
         if (fitres->status() != 0) { 
             Warning("StandardHypoTestInvDemo","Fit to the model failed - try with strategy 1 and perform first an Hesse computation");
